crowdin_api/api_resources/reports/resource.py

```diff
@@ -552,6 +553,124 @@ class ReportsResource(BaseReportsResource, BaseReportSettingsTemplatesResource):
     https://developer.crowdin.com/api/v2/#tag/Reports
     """
 
+    def get_report_archive_path(self, userId: int, archiveId: Optional[int] = None):
+        if archiveId is not None:
+            return f"users/{userId}/reports/archives/{archiveId}"
+
+        return f"users/{userId}/reports/archives"
```

Comment:
This implementation does not take into account the Crowdin Enterprise API difference discussed at https://github.com/crowdin/crowdin-api-client-python/issues/156#issuecomment-2134588954.

Reply:
I was planning to do the separate implementation of enterprise in the following resource I mean like write all enterprise APIs separately in this :
https://github.com/crowdin/crowdin-api-client-python/blob/a850ae3d80e104f3720cddf0de05d5bbd4fb6a7f/crowdin_api/api_resources/reports/resource.py#L723-L729

Reply:
@OmAximani0 okay, I'd suggest doing it together, within the current PR